gatorgrade/output/pyproject.toml

```diff
@@ -0,0 +1,12 @@
+[tool.poetry]
```

Comment:
It seems that perhaps `poetry add` or some similar command was run in `gatorgrade/output` -- this entire file structure seems pretty wonky. I think you need to take a step back and review where these changes should land. At minimum, there should not be a poetry installation in your `output` folder.

Reply:
Yes, we had it there before poetry was in the main branch and forgot to uninstall it. This has been fixed in my latest push!